player:
  ros__parameters:
    use_sim_time: false
    bag_uri: ""
    storage_id: "mcap"
    rate: 1.0
    loop: false
    start_paused: false
    topics: []
    publish_clock: false
    clock_frequency: 100.0

recorder:
  ros__parameters:
    use_sim_time: false
    output_uri: ""
    storage_id: "mcap"
    topics: []
    all_topics: false
    exclude_topics:
      - "/rosout"
      - "/parameter_events"
    max_bag_size: 0
    max_bag_duration: 0
